### ISSUE TITLE
[Question] PhantomJS

### ISSUE THREAD

**Issue by sakaia** [kubeflow/pipelines#3110]
The frontend uses PhantomJS. 
Regrettably, the PhantomJS will stop development.
Of course, kubernetes/pipelines used the PhantomJS in build time only.

Is there any plan to move from PhantomJS?
If so, which package you will plan to use?

Reference
- [frontend/Dockerfile](https://github.com/kubeflow/pipelines/blob/master/frontend/Dockerfile)
- [Archiving the project: suspending the development #15344](https://github.com/ariya/phantomjs/issues/15344)
- [update dockerfile and add build step of frontend #567](https://github.com/kubeflow/pipelines/pull/567)

**Comment by rmgogogo** [comment]
@Bobgy

**Comment by Bobgy** [comment]
Thanks @sakaia for raising the question.

I took this opportunity to investigate a little. Here's what I found:
* We used to use backstopjs for visual regression testing (which used phantomjs internally, but no longer now): https://github.com/kubeflow/pipelines/blob/master/frontend/backstop.ts
* But somehow after main maintainer for kfp frontend changed a few times, we simply don't know or use it any more.

Backstop js itself has removed phantomjs support in a later release: https://github.com/garris/BackstopJS/issues/772#issuecomment-456057756

We have a few options:
1. Restore backstop js tests
2. Just delete them

So far, I don't feel we have a strong need for visual regression testing, so I will go with deletion to simplify maintenance.
